bricks/advanced/src/data-providers/capture-snapshot.ts

```diff
@@ -0,0 +1,55 @@
+// istanbul ignore file: nothing logical except calling html2canvas.
+import { createProviderClass } from "@next-core/utils/general";
+import {
+  capture,
+  downloadImage,
+  getCanvasBlob,
+  uploadFile,
+} from "./snapshot.js";
+export type SnapshotOptions = {
+  /** 文件名称 */
+  name: string;
+  /** 文件类型 */
+  fileType: "image" | "pdf";
+  id?: string;
```

Comment:
建议改成 css selector，这样 id 可以传 `#id`，也可以支持其他的选择器

```suggestion
  selector?: string;
```